### ISSUE TITLE
The baseline of 3D-ResNet-18 on Mini-Kinetics-200 trained from scratch

### ISSUE THREAD

**Issue by GloryyrolG** [vt-vl-lab/SDN#8]
Hi Jinwoo,

Thanks for your work. One thing I want to know is, it seems the top-1 acc. of 3D-ResNet-18 on Mini-Kinetics-200 is not reported in the paper (or the reference paper). Could you kindly provide the information? And it is much better that you could share the .pth pretrained model of this.

Regards,

**Comment by jinwchoi** [comment]
The top-1 accuracy of 3D-ResNet-18 on Mini-Kinetics-200 is 56.32%. Hope this helps.
